modules/stanford_earth_capx/config/install/migrate_plus.migration.earth_capx_importer.yml

```diff
@@ -0,0 +1,150 @@
+uuid: 2200fcc9-747f-4e44-ae0d-b25826759bc3
```

Comment:
UUID should not exist in module install config.

Reply:
Fixed.

Reply:
Thanks.